Source/Scene/Cesium3DTile.js

```diff
@@ -187,10 +212,8 @@ define([
          */
         this.parentPlaneMask = 0;
 
-        this._debugBox = undefined;
-        this._debugcontentBox = undefined;
-        this._debugOrientedBoundingBox = undefined;
-        this._debugContentsOrientedBoundingBox = undefined;
```

Comment:
I kept only one debug volume for the tile and one debug volume for the tile contents. Not sure whether there is any need to keep them separated.